pages/reference/OS/extended-support-release.md

```diff
@@ -40,11 +40,11 @@ __Note:__ Once updated to an ESR version, it is not possible to update from an E
 
 ESR host OS versions are currently available for the following devices with additional device support planned:
 
-* Raspberry Pi (v1 and Zero)
+* Raspberry Pi (v1 / Zero / Zero W)
 * Raspberry Pi 3
```

Comment:
Not Raspberry Pi 2? ISTR hearing somewhere that some industrial customers still prefer the Pi 2 because of it's explicit lack of built-in wireless? (which I guess might be important for security and/or regulatory reasons)